model/lti.go

```diff
@@ -23,6 +23,7 @@ const (
 
 	LTI_USER_ID_PROP_KEY = "lti_user_id"
 
+	// plugin id of lti configuration plugin
```

Comment:
1. Comment over constant should start with constant name. 
2. mention what this plugin is used for.
e.g. LTI_PLUGIN_ID is the ID of the plugin used to maintain the LTI settings.